webapp/channels/src/components/more_direct_channels/list/list.tsx

```diff
@@ -107,11 +124,19 @@ const List = React.forwardRef((props: Props, ref?: React.Ref<MultiSelect<OptionV
             numRemainingText={
                 <FormattedMessage
                     id='multiselect.numPeopleRemaining'
-                    defaultMessage='Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '
+                    defaultMessage={'Use ↑↓ to browse, ↵ to select. You can\'t add more than 7 people. Please <a>create a channel</a> to include more people. '}
```

Comment:
The way this is implemented right now, we show this always. The idea is to show this only when we have reached the maximum number of people, right?

Reply:
Yes, you're right. I apologize for the misunderstanding. I will fix it.

Reply:
You can make up a new id. I guess `multiselect.maxPeople` should work.

Reply:
Thank you @larkox! I did it. Is it good practice or should I change something?